app/boards/shields/zmk_uno/zmk_uno.overlay

```diff
@@ -157,23 +157,26 @@
         toggle-mode;
 
         input-gpios
-            = <&arduino_header 4 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>
-            , <&arduino_header 3 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>
-            , <&arduino_header 2 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>
-            ;
+        = <&arduino_header 4 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>
+        , <&arduino_header 3 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>
+        , <&arduino_header 2 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>
+        ;
     };
 
     encoder: encoder {
         label = "ENCODER";
-        resolution = <4>;
+        steps = <80>;
         compatible = "alps,ec11";
-        a-gpios = <&arduino_header 14 GPIO_PULL_UP>;
-        b-gpios = <&arduino_header 15 GPIO_PULL_UP>;
+        a-gpios = <&arduino_header 15 GPIO_PULL_UP>;
+        b-gpios = <&arduino_header 14 GPIO_PULL_UP>;
     };
-
     sensors {
         compatible = "zmk,keymap-sensors";
         sensors = <&encoder>;
+        triggers-per-rotation = <20>;
+        left {
+            triggers-per-rotation = <20>;
+        };
```

Comment:
The config here allows setting a global value, or override the setting per sensor. Needed in case, say, you have a tactile encoder with a certain number of detents per rotation, and next to it a linear encoder that you want to trigger behaviors more frequently. This gets pulled in by the sensor code and leveraged by the keymap.

Reply:
I'm just confused by the overriding of a "left" encoder on a thing that only has one encoder as far as I can tell.

Reply:
For example and testing, more than requirement. Made it easy to comment out portions locally when testing this.